buildpacks/ruby/src/layers/bundle_install_layer.rs

```diff
@@ -353,9 +355,14 @@ fn bundle_install(env: &Env) -> Result<(), CommandError> {
         ],
     );
 
-    user::log_info(format!("\nRunning command:\n$ {command}"));
+    user::log_info(format!("\nRunning command:\n$ {command_string}"));
```

Comment:
I was thinking about making this chainable somehow.  Maybe use an extension trait on Command to implement some self yielding methods where we can add logging. For example `cmd_yield` where it yields itself but then returns itself.